.gitignore

```diff
@@ -2,3 +2,4 @@ build
 composer.lock
 docs
 vendor
+phpunit.xml
```

Comment:
Sure. Done. Isn't it though just the `phpunit.xml.dist` that is being commited?

P.S. You have a right to edit files on my branch :)